GitUI/Editor/RichTextBoxXhtmlSupportExtension.cs

```diff
@@ -33,7 +33,7 @@ static class RichTextBoxXhtmlSupportExtension
         private static int BeginUpdate(HandleRef handleRef)
         {
             // Prevent the control from raising any events.
-            int oldEventMask = NativeMethods.SendMessage(handleRef,
+            int oldEventMask = (int) NativeMethods.SendMessage(handleRef,
```

Comment:
:question: Doesn't it defeat the purpose of the `IntPtr` if you just cast it to an `int` anyway?